grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -324,7 +324,92 @@ public GrpcServiceBuilder addService(String path, BindableService bindableServic
             return addService(path, interceptor, methodDescriptor);
         }
         registryBuilder.addService(path, bindableService.bindService(), methodDescriptor,
-                                   bindableService.getClass());
+                                   bindableService.getClass(), null);
+        return this;
+    }
+
+    /**
+     * Adds a gRPC {@link BindableService} with functions to this {@link GrpcServiceBuilder}.
+     * The functions will be applied in the specified order and before functions decorated with the service.
+     * For more details, please refer to the following document.
+     *
+     * @see <a href="https://armeria.dev/docs/server-grpc#decorating-a-grpcservice">Decorating a GrpcService</a>
```

Comment:
Would it be a good idea to also copy this link to the javadocs of other methods which accept decorators?